packages/lib/services/searchengine/SearchEngine.ts

```diff
@@ -580,14 +575,32 @@ export default class SearchEngine {
 			// https://github.com/laurent22/joplin/issues/1075#issuecomment-459258856
 
 			try {
-				const { query, params } = queryBuilder(parsedQuery.allTerms);
+				const { query, params } = queryBuilder(parsedQuery.allTerms, true);
 				const rows = await this.db().selectAll(query, params);
 				this.processResults_(rows, parsedQuery);
 				return rows;
 			} catch (error) {
 				this.logger().warn(`Cannot execute MATCH query: ${searchString}: ${error.message}`);
 				return [];
 			}
+		} else {
+			searchString = this.normalizeText_(searchString);
+
+			let rows: any;
+			if (searchType === SearchEngine.SEARCH_TYPE_NONLATIN_SCRIPT) {
+				try {
+					const { query, params } = queryBuilder(parsedQuery.allTerms, false);
+					rows = await this.db().selectAll(query, params);
+				} catch (error) {
+					this.logger().warn(`Cannot execute LIKE query: ${searchString}: ${error.message}`);
+					return [];
+				}
```

Comment:
It looks like the same code as for SEARCH_TYPE_FTS expect with boolean true/false, so please refactor to avoid duplicate code.